lms/djangoapps/courseware/tests/test_masquerade.py

```diff
@@ -151,7 +153,7 @@ def verify_show_answer_present(self, show_answer_expected):
 
 
 @attr('shard_1')
-class NormalStudentVisibilityTest(MasqueradeTestCase):
+class NormalStudentVisibilityTest(MasqueradeTestCase, MilestonesTestCaseMixin):
```

Comment:
Same question regarding all these other test cases that now require `MilestonesTestCaseMixin`.